File: get_landing_stance.js
function getLandingStance(startStance, rotation) {
  const flips = Math.floor(Math.abs(rotation) / 180)
  if(flips % 2 === 0) {
    return startStance
  } else {
    return startStance === "Regular" ? "Goofy" : "Regular"
  }
}

File: get_landing_stance.py
import math as m
def get_landing_stance(start_stance, rotation):
    times = m.floor(abs(rotation) / 180 )
    if times % 2 == 0:
        return start_stance
    else:
        return "Goofy" if start_stance == "Regular" else "Regular"
    


